package.json

```diff
@@ -46,8 +46,8 @@
     "@biomejs/biome": "^1.7.0",
     "@tsconfig/strictest": "^2.0.5",
     "@types/node": "^20.12.7",
-    "tsup": "^8.0.2",
+    "tsup": "^8.4.0",
     "typescript": "^5.4.5",
-    "vitest": "^1.5.0"
+    "vitest": "^3.0.8"
```

Comment:
Although this is a major version bump, it's a test library. Tests have passed on the build so it's good